public/data/Chuksmbanaso.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Chuks",
+  "type": "personal",
+  "bio": "Fullstack Developer | Open Source Enthusiast",
```

Comment:
```suggestion
  "bio": "Full-stack Developer | Open Source Enthusiast",
```

Reply:
This is a personal preference, I have seen it written many ways and I also write it as 1 word `fullstack`